mojaloop-simulator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 description: "Helm Chart for the Mojaloop (SDK based) Simulator - mojaloop-simulator: v10.6.5; sdk-scheme-adapter: v10.5.3"
 name: mojaloop-simulator
-version: 10.6.0
-appVersion: "10.6.0"
+version: 11.0.0
+appVersion: "11.0.0"
```

Comment:
would the appVersion not be  10.6.3??

Reply:
Thanks for the review Rajiv.

Not really clear, there are three components here, `cache` (Redis), `mojaloop-simulator` and `sdk-scheme-adapter`. Helm is pretty non-specific how this should be used. It's also optional. I'd be somewhat inclined to remove it entirely. What do you think?

Reply:
so i would think we want to keep the scripts standardised, it its for multiple api changes then sure 11.0.0 should be fine